### PR TITLE
Convert all moveit_msg messages namespace

### PR DESCRIPTION
**Pull request by davetcoleman**
I wrote a python script to assist in the conversion

### CODE REVIEW

**Comment by vmayoral** [comment]
Same situation happening across different files.

**Comment by davetcoleman** [comment]
@vmayoral do you approve this PR?

File: moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h [hunk 1]
```diff
@@ -118,7 +118,7 @@ moveit_msgs::Constraints constructGoalConstraints(const robot_state::RobotState&
  *
  * @return A full constraint message containing both constraints
  */
-moveit_msgs::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
+moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::PoseStamped& pose,
```
Comment (vmayoral) [inline moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h]: PoseStamped use needs to be adapted according to https://index.ros.org/doc/ros2/Contributing/Migration-Guide/
```suggestion
moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::msg::PoseStamped& pose,
```

File: moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h [hunk 2]
```diff
@@ -136,7 +136,7 @@ moveit_msgs::Constraints constructGoalConstraints(const std::string& link_name,
...
```
Comment (vmayoral) [inline moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h]: Same here
```suggestion
moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_name, const geometry_msgs::msg::PoseStamped& pose,
```

**Comment by vmayoral** [comment]
You probably may need to review the script @davetcoleman

**Comment by davetcoleman** [comment]
As described in the title of the PR, this only addressed moveit_msg messages. Because it touches 182 files, I thought the scope was already complex enough. There are still many many message types that I have not yet converted, but they are outside the scope of this PR. 

The reason I didn't do all at once is because I had to manually find-replace every action and service based on exact name, because there's no way to easily tell if they are a message, service, or action

File: moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h (hunk 2 above)
Comment (davetcoleman) [reply moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h]: outside scope of this PR

File: moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h (hunk 1 above)
Comment (davetcoleman) [reply moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h]: outside scope of this PR

**Comment by davetcoleman** [comment]
> I don't think it should be merged in until after the open MoveIt1 PR's have been closed since it will likely break a lot of those

By this argument, we shouldn't be working on MoveIt 2 at all, but we are. I don't think there will be that many conflicts, actually.

> I would consider automating the change from #include <moveit_msgs/___.h> to #include <moveit_msgs/msg/___.h> in this same PR.

There are a lot of additional things that need to be done across the code base but I chose to keep this PR very simple as a demonstrator of how we can make brute-force changes across the code base. I do not have time to add any additional migration "features" to this PR so either we abandon it or we accept the progress it does make to the codebase.

File: moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h (hunk 2 above)
Comment (vmayoral) [reply moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/utils.h]: Note this approach is fundamentally different from the one we're following to get a minimum viable prototype (going module by module and ensuring that changes compile and make sense from a ROS2 perspective). Anyhow, contributions are valid and will save us time. 

Thanks for putting time into this.

**Comment by vmayoral** [comment]
@vmayoral approves this pull request.

Merging changes with our master branch of development, I can confirm that it compiles (at least it does in my local machine). I'd say we wait until https://travis-ci.org/AcutronicRobotics/moveit2/builds/502170373?utm_source=github_status&utm_medium=notification finishes (should take about an hour). Provided CI passes, I'd go forward with these changes.

**Comment by davetcoleman** [comment]
It passed.

**Pull request merged**